migration/README.md

```diff
@@ -0,0 +1,33 @@
+# Migration
+
+export_database.py contains script to migrate the default database from one instance of MongoDB to another.
+
+## Usage
+
+Script takes two default arguments:
+
+1. URI for source database using flag
+
+    `--source, -s`
+
+2. URI for target database using flag
+
+    `--target, -t`
+
+If the script encounters any problem during the migration with any of the documents, it pushes them to defaulters.bson<br/>
+If a document was already present in the target database, it pushes them to repeated_notices.bson
+
+If you want to further try to push from a bson file, specify it with argument
+<br/>`--try-defaulters or -d`
```

Comment:
You may want to update this section such that you let users of the script know that --source is only required when `--try-defaulters` is not being used.